scripts/main.cpp

```diff
@@ -138,7 +138,8 @@ int main(int argc, char** argv)
   if (ftThresholdHelper)
     ftThresholdHelper->init();
 
-  feedingDemo->getAda()->closeHand();
+  // TODO: uncomment this once we can send actions to hand
```

Comment:
I haven't had any issues even with this line commented out. Should we just leave it out (or if not, what does it add)?